src/pages/Video.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useVideo } from "../context/video-context";
+import "../style/video.css";
+import { useParams } from "react-router-dom";
+import { useLike } from "../context/like-context";
+import { usePlaylist } from "../context/playlist-context";
+import { useWatchLater } from "../context/watchlater-context";
```

Comment:
you can export it from index.js you have in context folder and import it in one line

Reply:
yes sure.